packages/web3-utils/src/utils.js

```diff
@@ -429,6 +429,10 @@ var sha3 = function (value) {
         value = hexToBytes(value);
     }
 
+    if (isBN(value)) {
+        value = value.toString(10);
```

Comment:
do 

```
if (isBN(value)) {
   value = value.toString()
}
```
as `BN` has a `toString` method

Reply:
+ i would probably do that at the start so if it needs to go into the first if it can 👍

Reply:
@joshstevens19 Since the facts that passed isBN means the value is an instance of BN. So value.toString(10) is calling of BN method.

Reply:
@joshstevens19 Since the facts that passed isBN means the value is an instance of BN. So value.toString(10) is calling of BN method.

Reply:
Read comments on thread if you want to know the conclusion 👍